example/src/ErrorModal/ErrorModal.tsx

```diff
@@ -14,13 +14,14 @@
  * limitations under the License.
  ***************************************************************************/
 import './ErrorModal.css'
+import { ReactElement } from 'react'
```

Comment:
For what do we need types from react?

Reply:
there is a warning about Missing return type on function